wordpress-mobile/WordPress-Android
# App strings not being translated for Indonesian and Hebrew

Fixes #18029 

To test:
1 - Install JP app;
2 - Tap the user picture to open "Me" screen;
3 - Tap on "App Settings";
4 - Tap on "Interface Language";
5 - Select Indonesian and navigate through the app: the strings should be translated to Indonesian;
6 - Repeat steps 2 - 4 and select Hebrew language. Navigate through the app: the strings should be translated to Hebrew.

Thanks @ParaskP7 for proposing the code changes on this fix.

<p float="left">
  <img src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/fbca485a-a0ba-4734-a9cf-ea5967e81c00" width="300" />
  <img src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/4b6551ab-a2d5-40cc-9b92-b30c25227560" width="300" /> 
</p>

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
`-`

PR submission checklist:


UI Changes testing checklist:
